javascripts/projectsService.js

```diff
@@ -218,10 +218,14 @@ define(['underscore'], function(_) {
       }
     }
 
-    var projects = _.map(ordering, function(i) {
+    var all_projects = _.map(ordering, function(i) {
       return _projectsData.projects[i];
     });
 
+    var projects = _.filter(all_projects, function(project) {
+      return project.stats ? project.stats['issue-count'] > 0 : true;
```

Comment:
This might be the most controversial change of the PR - hiding projects which have a cached `0` count from the list. I think this saves on users searching for projects that are currently inactive or don't have work,  and I'm open to refining this based on how users feel about this experience.

Reply:
@shiftkey This could be a good thing in many aspects. But also bad at few aspects such as when all the issues are closed for that repo and the owner or contributors forgets to mark the new issues with the label that they have given at up-for-grabs, then we will lose that Repo. The repo/ project could be good and contributors can reach the project and might open their own issues. It will be good to hide if the repo is not having any issues at all but to consider having no issues based on the label that they provided to us could be wrong. That's one bad aspect out of many could for your decision. Please weigh this with good aspects.

Reply:
For reference, there are 114 projects listed that currently have no open issues. I'm 👍 to refining this, but I'd like to get this caching work improvements out and we can then see how it feels for users.

> The repo/ project could be good and contributors can reach the project and might open their own issues.

Are they going to be doing this? That's not clear to me based on how users might be arriving at the site, and I worry listing lots of projects without tasks available makes it harder to find ideal projects for contributors.

> It will be good to hide if the repo is not having any issues at all but to consider having no issues based on the label that they provided to us could be wrong.

I've got a good grasp on detecting the incorrect label in https://github.com/up-for-grabs/up-for-grabs.net/pull/1330 which is now bundled into the "updated stats" action with a message like this:

```
Cannot find label 'Help wanted' for project 'matomo-org/matomo'
```

Resolving these requires manually checking the project to see if its due to a spelling mistake, a rename or whether no suitable candidate exists. This could definitely be tweaked based on feedback and potential additional data we need, but it's feeling pretty reliable at the moment.

Reply:
Sure, we can do that. Thanks for the explanation. Things seem valid. Let's push it